Make.inc

```diff
@@ -338,6 +338,10 @@ CXX := $(CROSS_COMPILE)g++
 JCFLAGS := -std=gnu99 -pipe $(fPIC) -fno-strict-aliasing -D_FILE_OFFSET_BITS=64
 JCPPFLAGS :=
 JCXXFLAGS := -pipe $(fPIC) -fno-rtti
+ifneq ($(OS), WINNT)
+# Do no enable on windows to avoid warnings from libuv.
```

Comment:
From a previous AppVeyor run, [L794](https://ci.appveyor.com/project/StefanKarpinski/julia/build/1.0.12937/job/f2h04rxavrcemlaa#L794) to [L2133](https://ci.appveyor.com/project/StefanKarpinski/julia/build/1.0.12937/job/f2h04rxavrcemlaa#L2133).